libraries/AP_BattMonitor/AP_BattMonitor_UAVCAN.cpp

```diff
@@ -21,6 +21,17 @@
 
 extern const AP_HAL::HAL& hal;
 
+const AP_Param::GroupInfo AP_BattMonitor_UAVCAN::var_info[] = {
+
+    // @Param: CUR_ADJUST
+    // @DisplayName: Adjusts current scale
```

Comment:
Let's try and put Battery at the front of the description..

Reply:
maybe "Battery scale multiplier"